Makefile

```diff
@@ -24,7 +24,7 @@ prepare-build: install
 	cp "${GOPATH}"/bin/broker build/broker
 
 build: prepare-build
-	docker build ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}
+	docker build -t ${BROKER_IMAGE}:${TAG} ${BUILD_DIR}
```

Comment:
@fabianvf @eriknelson I think this needs to be rebased and changed under build-image now to:
`docker build -f ${BUILD_DIR}/Dockerfile-src -t ${BROKER_IMAGE}:${TAG} ${BUILD_DIR}`